src/main/java/io/jenkins/plugins/analysis/core/scm/BlameFactory.java

```diff
@@ -20,6 +22,20 @@
  * @author Lukas Krose
  */
 public final class BlameFactory {
+
+    private static JenkinsFacade jenkinsFacade = new JenkinsFacade();
+    private static GitChecker gitChecker = new GitChecker();
```

Comment:
While in general this approach is correct, in our case using a field for `GitChecker` is not possible. This will raise a `ClassNotFoundException` if the Git plugin is not installed. So move `GitChecker` back to the old position. You can't replace it with a stub.